charts/newrelic-logging/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart to deploy New Relic Kubernetes Logging as a DaemonSet, supporting both Linux and Windows nodes and containers
 name: newrelic-logging
-version: 1.23.5
+version: 1.23.6
```

Comment:
Why is the version bumped up to 1.23.6. We want to have a version suffixed with something like "-beta" to the version name. And we should tag this helm chart with the same. So as to be able to install it with something like **helm install --version "some_version-beta"**